svjis/articles/management/commands/svjis_setup.py

```diff
@@ -120,15 +125,11 @@ def create_groups():
     print("Done")
 
 
-def create_admin_user():
+def create_admin_user(password: str = None):
     print("Creating admin user...")
-    u = User.objects.create(
-        username='admin', email='admin@test.cz', password=make_password('masterkey'), last_name='admin'
+    u = User.objects.create_superuser(
+        username='admin', email='admin@test.cz', password=make_password(password), last_name='admin'
```

Comment:
Zdá se, že metoda `create_superuser` očekává `password` v otevřeném formátu. Použití `make_password` pak způsobí dvojité zahešování a nefunkčnost uživatelem zadaného hesla.

Reply:
good find. díky za opravu!